### ISSUE TITLE
Lack of most shadows

### ISSUE THREAD

**Issue by SweepSwap** [IrisShaders/Iris#237]
title says it all 
![image](https://user-images.githubusercontent.com/77552328/114348171-2ec1f580-9b34-11eb-88c5-b6d5a9a3a97d.png)

**Comment by SweepSwap** [comment]
this was on the sodium fork but it was the same when i changed to the normal branch, just with half the fps

**Comment by coderbot16** [comment]
Duplicate of https://github.com/IrisShaders/Iris/issues/84. The shadow code is incomplete and is still being worked on, it is not yet public. Once it is ready it will be made publicly available.
